products/powershell.md

```diff
@@ -15,15 +15,15 @@ releases:
 -   releaseCycle: "7.3"
     lts: false
     eol: false
-    latest: "7.3.1"
-    latestReleaseDate: 2022-12-13
+    latest: "7.3.2"
```

Comment:
Latest versions are not in the changelogs, maybe they switched to GitHub releases (https://github.com/PowerShell/PowerShell/releases/tag/v7.3.2) ?

Reply:
Yes, it looks like they have move to releases. -> Created #2372

Reply:
Ok, thanks @BiNZGi. Resolving this conversation (now there is a dedicated issue).